src/google_map.js

```diff
@@ -663,6 +662,10 @@ export default class GoogleMap extends Component {
           this.heatmap.setMap(map);
         }
 
+        maps.event.addListener(map, 'tilesloaded', () => {
```

Comment:
Should we add the listener all the times, or only when the user has set an onTilesLoaded prop? What do you think?